pkg/certs/types.go

```diff
@@ -6,10 +6,21 @@ import (
 	"github.com/openshift/library-go/pkg/certs/cert-inspection/certgraphapi"
 )
 
+// PKIRegistryInfo holds information about TLS artifacts stored in etcd. This includes object location and metadata based on object annotations
+type PKIRegistryInfo struct {
```

Comment:
Can this type be entirely contained to origin since we don't produce it raw and we're going to generate docs from here?

Reply:
We could `PKIList` there instead